mongodb/src/test/java/org/hibernate/ogm/datastore/mongodb/test/dialectinvocations/GridDialectOperationInvocationsTest.java

```diff
@@ -121,8 +121,6 @@ public void insertGetOfEntityAndElementCollectionAndDelete() throws Exception {
 				"getAssociation",
 				"getTuple",
 				"getAssociation",
-				"removeAssociation",
```

Comment:
I suppose they are part of the subsequent `executeBatch()`? I might be wrong, but there should be a way for asserting the batch contents, too (if not we should create such testing facility as part of a separate issue).